components/widgets/menu/menu_items/menu_item_external_link.tsx

```diff
@@ -17,7 +17,9 @@ export const MenuItemExternalLinkImpl: React.FC<Props> = ({url, text, onClick}:
         href={url}
         onClick={onClick}
     >
-        {text}
+        <span className='MenuItem__primary-text'>
```

Comment:
I don't know if this is right, but this is going to affect to any `ExternaLink` used, that can appear in other contexts.

Reply:
Thanks for flagging! This fixes a visual inconsistency between external links and all other links.

I did double check all uses of the external menu item throughout the webapp and in each case, the buttons function as expected.